Description: unset CMAKE_*_FLAGS to avoid overriding build hardening flags
Author: Vincent Cheng <vcheng@debian.org>
Forwarded: not-needed
Last-Update: 2020-08-16

--- a/cmake/ConkyBuildOptions.cmake
+++ b/cmake/ConkyBuildOptions.cmake
@@ -37,12 +37,6 @@
 endif(NOT CMAKE_BUILD_TYPE)
 
 # -std options for all build types
-set(CMAKE_C_FLAGS "-std=c99"
-    CACHE STRING "Flags used by the C compiler during all build types."
-    FORCE)
-set(CMAKE_CXX_FLAGS "-std=c++17"
-    CACHE STRING "Flags used by the C++ compiler during all build types."
-    FORCE)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS ON)
